src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -343,6 +343,7 @@ typedef struct {
     int event_id;               /* fixed field, do not move */
     MPIR_Request *parent;       /* Parent request           */
     void *buf;
+    size_t offset;
```

Comment:
Please use `MPI_Aint` here instead of `size_t`.

Reply:
changed.